src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
```

Comment:
```suggestion
<footer class="footer">
```
可能なところでは[セマンティック要素](https://developer.mozilla.org/ja/docs/Glossary/Semantics#%E3%82%BB%E3%83%9E%E3%83%B3%E3%83%86%E3%82%A3%E3%83%83%E3%82%AF%E8%A6%81%E7%B4%A0)を使っていきましょう！